md/tests/test_views.py

```diff
@@ -59,3 +60,34 @@ def test_agency_list_sorted_agencies(self):
             self.assertEqual(2, len(chunks[0]))
             self.assertEqual(2, len(chunks[1]))
             self.assertEqual(1, len(chunks[2]))
+
+
+class TestSearchView(TestCase):
+    def test_search_good_data(self):
+        self.factory = RequestFactory()
+        request = self.factory.get('/md/search', data={
```

Comment:
just use "factory" in these two lines instead of "self.factory"; you don't need to save the new factory for other methods to access